src/avg.jl

```diff
@@ -21,20 +21,10 @@ end
 
 type AvgCutPruner{N, T} <: AbstractCutPruner{N, T}
```

Comment:
Thanks ^^ `cuts_DE` meant `D` -> optimality and `E` feasibility (or the opposite I don't remember). Now it does not make sense anymore so `A` is more agnostic :-D